exec/src/coqnum.ml

```diff
@@ -25,13 +25,10 @@ open Constr
 let int_of_coqpositive =
```

Comment:
That may be obvious but we should probably indicate that this function does not handle overflow.